ci-operator/step-registry/telco-bastion/setup/telco-bastion-setup-commands.sh

```diff
@@ -14,24 +14,25 @@ cat << EOF > ~/inventory
 sshd.bastion-telco ansible_ssh_user=tester ansible_ssh_common_args="-o ConnectTimeout=5 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null" ansible_ssh_private_key_file=/var/run/ssh-pass
 EOF
 
-# We will switch to kcli in the near future
 cat << EOF > ~/ocp-install.yml
 ---
-- name: Prepare to run ansible-ipi-install playbook
+- name: Grab and run kcli to install openshift cluster
   hosts: all
   tasks:
   - name: Clone repo
     git:
-      repo: https://github.com/openshift-kni/baremetal-deploy.git
-      dest: "~/baremetal-deploy"
+      repo: https://github.com/karamb/kcli-openshift4-baremetal.git
+      dest: "~/kcli-openshift4-baremetal"
       version: master
       force: yes
     retries: 5
+  - name: Remove old installer vm
+    shell: kcli delete vm --yes cnfdc5-installer
   - name: Run deployment
-    shell: ansible-playbook -i /home/tester/inventory playbook.yml
+    shell: kcli create plan --paramfile /home/tester/kcli_parameters.yml upstream_ci --wait
     args:
-      chdir: ~/baremetal-deploy/ansible-ipi-install
-  - name: Run playbook to copy kubeconfig from provisionhost vm to bastion vm
+      chdir: ~/kcli-openshift4-baremetal
+  - name: Run playbook to copy kubeconfig from installer vm to bastion vm
     shell: ansible-playbook -i /home/tester/inventory /home/tester/kubeconfig.yml
```

Comment:
that can be done now with kcli too
assuming installation was successful, 
kubeconfig is always under
/root/ocp/auth/kubeconfig
on the installer..

Reply:
Yeah the kubeconfig.yml playbook leverages `kcli ssh`. Are you saying you want me to copy the playbook's tasks here instead? I have to double check that nothing is internal only (there were internal things in the ansible-ipi-install scenario)